test/dub_package/source/app.d

```diff
@@ -43,3 +43,119 @@ unittest
     scope parser = new Parser!ASTBase(null, null, false);
     assert(parser !is null);
 }
+
+
+// inits the DMD compiler
+void init()
+{
+    import ddmd.globals : global;
+
+    global._init;
+    global.params.isLinux = 1;
+    import ddmd.dmodule : Module;
+    import ddmd.id : Id;
+    import ddmd.mtype : Type;
+    import ddmd.target : Target;
+    import ddmd.expression : Expression;
+    import ddmd.objc : Objc;
+    import ddmd.builtin : builtin_init;
+
+    // Initialization
+    Type._init();
+    Id.initialize();
+    Module._init();
+    Target._init();
+    Expression._init();
+    Objc._init();
+    builtin_init();
+}
+
+// add import paths
+void addImports(string[] path)
+{
+    import ddmd.globals : global;
+    import ddmd.arraytypes : Strings;
+
+    Strings* res = new Strings();
+    foreach (p; path)
+    {
+        import std.string : toStringz;
+        Strings* a = new Strings();
+        a.push(p.toStringz);
+        res.append(a);
+    }
+    global.path = res;
+}
+
+// frontend
+unittest
+{
+    import ddmd.astcodegen : ASTCodegen;
+    import ddmd.dmodule : Module;
+    import ddmd.globals : Loc;
+    import ddmd.parse : Parser;
+    import ddmd.statement : Identifier;
+    import ddmd.tokens : TOKeof;
+    import ddmd.id : Id;
+
+    init;
+    addImports(["/usr/include/dlang/dmd/"]);
```

Comment:
This is the worst bit about this PR. While, for this test we could mock `object.d`, I think we should show potential users how they can add import paths as they most likely want to do so.
Ideas:
- mock `object.d`
- use sth. like `../../../druntime/import` (i.e. depend on existing repo layout)
- call DMD and search config strings
- refactor config parsing from `mars.d`, s.t. it can be used here